demo/standalone/Vagrantfile

```diff
@@ -17,8 +17,7 @@ VAGRANTFILE_API_VERSION = "2"
 $channel = "alpha"
 $nodes = 3
 $new_discovery_url="https://discovery.etcd.io/new?size=1"
-$node_disks = 3
-$disk_size = 5
+$node_disk_sizes = [5, 5, 5] # size in GB of each disk
```

Comment:
we can do that...i was testing with `[5,6,7]` until switching it back to 3 5's before committing so that I would maintain the current setup. it might be nice to always be testing with a heterogenous environment though.

although that would probably have bigger impact in the coreos-kubernetes repo since we use kubernetes more often than standalone vagrant.

what sizes do you think we should use by default?